cmdstanpy/stanfit.py

```diff
@@ -1635,3 +1917,27 @@ def from_csv(
             'An error occured processing the CSV files:\n\t{}'.format(str(e))
         ) from e
     return fit
+
+
+def add_var_to_xarray(
+    data: Dict[str, xr.DataArray],
```

Comment:
This type is incorrect I believe - it seems like the dictionary is `Dict[str, Tuple[Tuple[str,...], np.ndarray]`. It's not terribly far off from an xrarry DataArray object (you have everything needed to instantiate one) but it isn't actually calling xr.DataArray (which is a good thing!)

Reply:
fixed